### PR TITLE
Entertainment monitor screen now changes if there is any video camera active

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Entertainment monitor screen now changes if there is any video camera active.
Added a non live icon screen to entertainment monitor.
Video Camera now has a 2 SECOND cooldown on usage.
## Why It's Good For The Game

This change would let the crew know if there are any active cameras to view on the entertainment monitors, which would encourage their use.
## Images of changes

https://user-images.githubusercontent.com/77684085/209743163-25ed0e4b-70d3-4584-8cf1-5984248b8f83.mp4

## Testing

- Used the video camera looking for entertainment monitor changes
- Tested the behaviour when more than one camera is active
## Changelog
:cl:
tweak: Entertainment monitor screen now changes if there is any video camera active
tweak: Video Camera now has a 2 SECOND cooldown on usage
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/camera.dm [hunk 1]
```diff
@@ -214,17 +214,25 @@
...
 /obj/machinery/computer/security/telescreen/entertainment/Initialize()
 	. = ..()
 	set_light(1, LIGHTING_MINIMUM_POWER) //so byond doesnt cull, and we get an emissive appearance
 
+/obj/machinery/computer/security/telescreen/entertainment/update_overlays()
+	if(feeds_on)
+		icon_screen = "entertainment"
+	else
+		icon_screen = "entertainment_off"
+	. = ..()
```
Comment (lewcc) [inline code/game/machinery/computer/camera.dm:234]:
```suggestion
	return ..()
```

File: code/game/machinery/computer/camera.dm [hunk 2]
```diff
@@ -214,17 +214,25 @@
...
 	desc = "Damn, they better have Paradise TV on these things."
 	icon_state = "entertainment_console"
-	icon_screen = "entertainment"
+	icon_screen = "entertainment_off"
 	light_color = "#FFEEDB"
 	light_range_on = 0
 	network = list("news")
 	luminosity = 0
 	circuit = null
+	var/feeds_on = 0
```
Comment (lewcc) [inline code/game/machinery/computer/camera.dm:223]: Can you add an autodoc comment for this variable? Like
```
/// What it does
var/feeds_on = 0
```

File: code/modules/paperwork/photography.dm [hunk 3]
```diff
@@ -553,6 +554,7 @@ GLOBAL_LIST_INIT(SpookyGhosts, list("ghost","shade","shade2","ghost-narsie","hor
 	w_class = WEIGHT_CLASS_NORMAL
 	materials = list(MAT_METAL = 1000, MAT_GLASS = 500)
 	var/on = FALSE
+	var/video_cooldown
```
Comment (Sirryan2002) [inline code/modules/paperwork/photography.dm:557]:
```suggestion
	var/video_cooldown = 0
```

File: code/modules/paperwork/photography.dm [hunk 4]
```diff
@@ -571,8 +573,18 @@ GLOBAL_LIST_INIT(SpookyGhosts, list("ghost","shade","shade2","ghost-narsie","hor
 		camera.c_tag = null
 		QDEL_NULL(camera)
 	visible_message("<span class='notice'>The video camera has been turned [on ? "on" : "off"].</span>")
+	for(var/obj/machinery/computer/security/telescreen/entertainment/TV in GLOB.machines)
+		if(on)
+			TV.feeds_on += 1
```
Comment (Sirryan2002) [inline code/modules/paperwork/photography.dm:578]:
```suggestion
			TV.feeds_on++
```

File: code/modules/paperwork/photography.dm [hunk 5]
```diff
@@ -571,8 +573,18 @@ GLOBAL_LIST_INIT(SpookyGhosts, list("ghost","shade","shade2","ghost-narsie","hor
...
+		else
+			TV.feeds_on -= 1
```
Comment (Sirryan2002) [inline code/modules/paperwork/photography.dm:580]:
```suggestion
			TV.feeds_on--
```

**Review by Sirryan2002 — APPROVED** [review]

**Pull request merged**